autoinstall/Ubuntu/Flutter/user-data.j2

```diff
@@ -0,0 +1,54 @@
+#cloud-config
+autoinstall:
+  version: 1
+  early-commands:
+    - echo '{{ autoinstall_start_msg }}' >/dev/ttyS0
+  locale: en_US.UTF-8
+  keyboard:
+    layout: us
+  network:
+    version: 2
+    ethernets:
+      all-eth:
+        match:
+          name: "ens*"
```

Comment:
The ethernet names might not be started with 'ens'.

Reply:
During the autoinstall, the ifname is ens*.  
The installer will overwrite the file with default configuration as follows after finish autoinstall.
network:
    version: 2
    render: NetworkManager

Thanks
Yuhua Zou